docs/topics/patching-nodes.md

```diff
@@ -0,0 +1,41 @@
+# Patching Cluster Nodes
+
+To favor a consistent behavior, AKS Engine's approach to infrastructure management is immutability.
+By default, cluster nodes are not patched on bootstrap nor on a regular basis. 
```

Comment:
`linuxProfile.enableUnattendedUpgrade` is set to `"true"` by default if not specified. https://github.com/Azure/aks-engine-azurestack/blob/master/docs/howto/troubleshooting.md#prevent-unattended-upgrades